src/OSPSuite.UI/Views/Importer/ImporterView.cs

```diff
@@ -29,7 +29,18 @@ public ImporterView()
 
       public void ShowErrorMessage(string message)
       {
-         XtraMessageBox.Show(this, message, "", MessageBoxButtons.OK, MessageBoxIcon.Error);
+         var args = new XtraMessageBoxArgs
```

Comment:
Why is there a call to the user in the view? THis is not ok and was missed before. The presenter is the one that needs to manager usser interaction. Never the view

Reply:
yeap, I also agree, is what I was saying in the description of the PR. I will change this accordingly then.

Reply:
done